src/utils/ADempiere/constants/report.js

```diff
@@ -32,3 +34,56 @@ export const staticReportRoutes = [
     parametersList: [{ columnName: 'ValidFrom', value: today }, { columnName: 'MustBeStocked', value: false }]
   }
 ]
+
+export const reportExportTypes = [
```

Comment:
Rename the constant to upper case `REPORT_EXPORT_TYPES`

Reply:
[[95ebee3](https://github.com/solop-develop/frontend-core/pull/2060/commits/95ebee31ba8310075bb2f93da62a0008deccb6ab)](url)